examples/volume_RT_shim_respiratory.py

```diff
@@ -0,0 +1,56 @@
+# Context: That scripts covers a volume-wise realtime shimming with respiratory probe scenario 
+#
+# Sort unorganized images from DICOM socket transfer
+# dcm2bids is used to create folder structure and nifti conversion
+# unsortedDicomDir: path to the folder containing the dicoms
+# niftiPath: path where we want dcm2bids to store the nifti files
+# 
+# > dicom_to_nifti( unsortedDicomDir, niftiPath )
+#
+# Load images from the path
+# Using niftiPath will prompt the user to select the appropriate acquisition. Alternatively, if the path to the acquisition is known, the path can be changed and the function won't prompt the user
```

Comment:
> `Using niftiPath will prompt the user to select the appropriate acquisition. `

this is inconsistent with the variable niftiPath introduced above

Reply:
I don't think it is because we defined niftiPath as the path were the files are stored (so it corresponds to the folder). When we use that folder as input, the user must precise which acquisition corresponds to the magnitudes or the phases. But if the direct path to the acquisition is given, the user's intervention won't be needed. @po09i can you confirm ?

Reply:
clarification is needed here: what is meant by "path", what is meant by "acquisition"?
example of folder structure should easily clarify

Reply:
Yes this is how it works @gaspardcereza, maybe add some clarification as Julien suggested to make things clearer